Update .ebextensions (add Cloudwatch agent & AWS Inspector) (#14718)
metabase/metabase: docs/operations-guide/running-metabase-on-elastic-beanstalk.md

## Quick Launch

> Download the [Metabase Community Edition AWS source bundle file](https://downloads.metabase.com/{{ site.latest_version }}/metabase-aws-eb.zip) to upload to Elastic Beanstalk. 
> 
> If you are looking to deploy our **Enterprise Edition**, you need to go to the link https://downloads.metabase.com/enterprise/**version**/metabase-aws-eb.zip. replacing the **version** part of the URL with the version that you want to deploy (e.g v1.38.3).

Reviewer: ```suggestion
Download the [Metabase Open Source Edition AWS source bundle file](https://downloads.metabase.com/{{ site.latest_version }}/metabase-aws-eb.zip) to upload to Elastic Beanstalk. 

For the **Enterprise Edition**, visit 
https://downloads.metabase.com/enterprise/**version**/metabase-aws-eb.zip,

replacing the **version** part of the URL with the version that you want to deploy (e.g., v1.38.3).
```

Is there instead a landing page we could direct them to, rather than ask people to choose a version and compose a URL?

Reviewer 2: Unfortunately no, I'll delete these lines and if the customers ask, we'll send it through the support channel
